reference/tiflash/maintain.md

```diff
@@ -58,8 +58,47 @@ category: reference
 > **注意：**
 >
 > 如果在集群中所有的 TiFlash 节点停止运行之前，没有取消所有同步到 TiFlash 的表，则需要手动在 PD 中清除同步规则，否则无法成功完成 TiFlash 节点的下线。
->
-> 手动在 PD 中清除同步规则的方法是：发送 DELETE 请求 `http://<pd_ip>:<pd_port>/pd/api/v1/config/rule/tiflash/<rule_id>`，rule_id 是需要清除的 rule 的 id。
+
+手动在 PD 中清除同步规则的步骤如下：
+
+1. 查询当前 PD 实例中所有与 TiFlash 相关的的数据同步规则。
```

Comment:
```suggestion
1. 查询当前 PD 实例中所有与 TiFlash 相关的数据同步规则。
```

Reply:
Can you open a PR to fix this typo? @lidezhu